Azure/sonic-buildimage
# Download picocom version 3.1-2 from stretch-backports; No longer build from source

**DO NOT MERGE YET**

**- What I did**

Now that we have moved up from Debian Jessie to Stretch, there is no need to build Picocom version 3.1-2 from source, as it is available from stretch-backports.

**- How I did it**

This PR essentially reverts https://github.com/Azure/sonic-buildimage/pull/1943 and downloads the package from stretch-backports instead of stretch.

**- How to verify it**

Build and install an image, then ensure picocom version 3.1-2 is installed using `dpkg -l`
